quickstart/nodejs/src/index.ts

```diff
@@ -0,0 +1,86 @@
+import { DefaultAzureCredential } from '@azure/identity';
+import { EntraIdCredentialsProviderFactory, REDIS_SCOPE_DEFAULT } from '@redis/entraid';
+import { createCluster, RedisClusterType, RedisModules, RedisFunctions, RedisScripts } from '@redis/client';
+import * as net from 'node:net';
+
+const redisEndpoint = process.env.REDIS_ENDPOINT!;
+if (!redisEndpoint) {
+    console.error('REDIS_ENDPOINT is not set. It should look like: `cache-name.region-name.redis.azure.net:<PORT>`. Find the endpoint in the Azure portal.');
+    process.exit(1);
+}
+
+const [redisHostName, _] = redisEndpoint.split(":");
+
+let client;
+
+function createConnection(): RedisClusterType<RedisModules, RedisFunctions, RedisScripts>  {

```

Comment:
Sorry, let's name this `createRedisClient` since we don't actually call `.connect()` in here